e2e/cypress/integration/team_settings/invite_members_spec.js

```diff
@@ -10,11 +10,11 @@
 // Stage: @prod
 // Group: @team_settings
 
-import users from '../../fixtures/users.json';
+// import users from '../../fixtures/users.json';
```

Comment:
Can we remove these?

Reply:
Done, included in https://github.com/mattermost/mattermost-webapp/pull/5723/commits/0ad2906e92a0a04f0c5b8d7724819a5b7f517a22